geo/parse.go

```diff
@@ -16,14 +16,19 @@ package geo
 
 import (
 	"reflect"
+	"strconv"
 	"strings"
+
+	"github.com/mmcloughlin/geohash"
 )
 
 // ExtractGeoPoint takes an arbitrary interface{} and tries it's best to
 // interpret it is as geo point.  Supported formats:
 // Container:
 // slice length 2 (GeoJSON)
 //  first element lon, second element lat
+// string (coordinates seperated by comma, or a geohash)
+//  first element lat, second element lon
```

Comment:
minor extra space? :)

Reply:
That extra space before first, is following the format from before.

Per elasticsearch spec here: https://www.elastic.co/guide/en/elasticsearch/reference/current/geo-point.html

With slices, its lon first and lat after, but with strings its lat first and lon after - that's what the comment(s)^ above talk about.